File: back/ProEventos/src/ProEventos.Domain/Models/Palestrante.cs
namespace ProEventos.Domain.Models;

public class Palestrante : BaseEntity
{
    public string Nome { get; set; }
    public string MiniCurriculo { get; set; }
    public string ImagemURL { get; set; }
    public string Telefone { get; set; }
    public string Email { get; set; }
    public IEnumerable<RedeSocial> RedesSociais { get; set; }
    public IEnumerable<PalestranteEvento> PalestrantesEventos { get; set; }
}

File: front/ProEventos-App/src/app/models/palestrante.ts
import { Evento } from "./evento";
import { RedeSocial } from "./redeSocial";

export interface Palestrante {
    id: number;
    nome: string;
    miniCurriculo: string;
    imagemURL: string;
    telefone: string;
    email: string;
    redesSociais: RedeSocial[];
    palestranteEventos: Evento[];
}
